doc/NOW-spec.md

```diff
@@ -743,7 +784,6 @@ The NOW_EXEC_MSG message is used to execute remote commands or scripts.
 
 | Value | Meaning |
 |-------|---------|
-| NOW_EXEC_CAPSET_MSG_ID<br>0x00 | NOW_EXEC_CAPSET_MSG |
```

Comment:
question: Are we removing this message? Is it superseded by something else?

Reply:
Yep. This was previosuly in `exec` message class -- now it was moved to `channel` message class